content/de/docs/setup/_index.md

```diff
@@ -27,7 +27,7 @@ Sie sollten eine lokale Lösung auswählen, wenn Sie:
 * Zum ausprobieren oder wenn Sie etwas über Kubernetes lernen möchten
 * Cluster lokal entwickeln und testen
 
-Wählen Sie eine [lokale Lösung](/docs/setup/pick-right-solution/#local-machine-solutions) aus.
+Wählen Sie eine [lokale Lösung](/de/docs/setup/) aus.
```

Comment:
Can you avoid localized links, i.e.  `/docs/setup` instead of `/de/docs/setup` and put the anchors like `/docs/setup/#gehostete-lösungen` please?

Reply:
I'm not sure yet how we can best use localized links, e.g. `/docs/setup/` vs `/de/docs/setup/`. If someone has an idea please pitch in. Otherwise I'll have a look at it the next couple of days.